ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -24,11 +23,11 @@ public class EpisodeModel : ValidationModel
         [BindProperty]
         public int NotificationId { get; set; }
         
-
-        public EpisodeModel(INotificationService service, NtbsContext context)
+        public EpisodeModel(INotificationService service, NtbsContext context, IAuditService auditService)

```

Comment:
I decided it made more sense to delegate the auditing to a specific service rather than performing auditing in a base class, as otherwise we would need to pass the auditDbContext through all the way down the inheritance chain so it can be properly DI-ed

Reply:
Yeah, makes sense!